extensions/amp-mustache/0.2/test/test-amp-mustache.js

```diff
@@ -110,6 +110,41 @@ describes.repeated('amp-mustache 0.2', {
         .equal('<a href="javascript:alert(0)">abc</a>');
   });
 
+  describe('custom delimiters', () => {
+    it('should require beginning and ending delimiter', () => {
+      templateElement.setAttribute('data-custom-delimiters', '<%');
+      innerHtmlSetup(
```

Comment:
I can't comment on the appropriate line: https://github.com/ampproject/amphtml/blob/7981c6743bc5712e18ec281b185a54cba2dd7a95/extensions/amp-mustache/0.1/test/test-amp-mustache.js#L69

I believe is incorrect, you need to insert text into the script element using `innerHTML`.